robottelo/hosts.py

```diff
@@ -1551,6 +1563,9 @@ def cli(self):
 
 
 class Satellite(Capsule, SatelliteMixins):
+    product_rpm_name = 'satellite'
```

Comment:
What do reviewers think?
```suggestion
    downstream_rpm_name = 'satellite'
```

Reply:
either way is fine